docs/cudf/source/user_guide/changes.md

```diff
@@ -0,0 +1,612 @@
+Out[3]:
+0   2000-01-01 00:00:00.012345
+1   2000-01-31 00:00:00.012345
+2   2000-02-29 00:00:00.012345
+dtype: datetime64[us]
+```
+
+### Change in bitwise operation results
+
+
+Bitwise operations between two in-equally indexed objects will now not result in boolean results.
+
+
+Old behavior:
+
+```python
+In [1]: import cudf
+
+In [2]: import numpy as np
+
+In [3]: s = cudf.Series([1, 2, 3])
+
+In [4]: p = cudf.Series([10, 11, 12], index=[2, 1, 10])
+
+In [5]: np.bitwise_or(s, p)
+Out[5]:
+0      True
+1      True
+2      True
+10    False
+dtype: bool
+```
+
+New behavior:
+```python
+In [5]: np.bitwise_or(s, p)
+Out[5]:
+0     <NA>
+1       11
+2       11
+10    <NA>
+dtype: int64
+```
+
+
+### ufuncs will perform re-indexing
+
+Performing a numpy ufunc operation on two objects with mismatching index will result in re-indexing:
+
+Old behavior:
+
+```python
+In [1]: import cudf
+
+In [2]: df = cudf.DataFrame({"a": [1, 2, 3]}, index=[0, 2, 3])
+
+In [3]: df1 = cudf.DataFrame({"a": [1, 2, 3]}, index=[10, 20, 3])
+
+In [4]: import numpy as np
+
+In [6]: np.add(df, df1)
+Out[6]:
+   a
+0  2
+2  4
+3  6
+```
+
+
+New behavior:
+
+```python
+In [6]: np.add(df, df1)
+Out[6]:
+       a
+0   <NA>
+2   <NA>
+3      6
+10  <NA>
+20  <NA>
+```
+
+
+### `DataFrame` vs `Series` comparisons need to have matching index
+
+Going forward any comparison between `DataFrame` & `Series` objects will need to have matching indexe's, i.e., the column names of `DataFrame` should match index of `Series`:
+
+Old behavior:
+```python
+In [1]: import cudf
+
+In [2]: df = cudf.DataFrame({'a':range(0, 5), 'b':range(10, 15)})
+
+In [3]: df
+Out[3]:
+   a   b
+0  0  10
+1  1  11
+2  2  12
+3  3  13
+4  4  14
+
+In [4]: s = cudf.Series([1, 2, 3])
+
+In [6]: df == s
+Out[6]:
+       a      b      0      1      2
+0  False  False  False  False  False
+1  False  False  False  False  False
+2  False  False  False  False  False
+3  False  False  False  False  False
+4  False  False  False  False  False
+```
+
+New behavior:
+```python
+In [5]: df == s
+
+ValueError: Can only compare DataFrame & Series objects whose columns & index are same respectively, please reindex.
+
+In [8]: s = cudf.Series([1, 2], index=['a', 'b'])
+# Create a series with matching Index to that of `df.columns` and then compare.
+
+In [9]: df == s
+Out[9]:
+       a      b
+0  False  False
+1   True  False
+2  False  False
+3  False  False
+4  False  False
+```
+
+
+### `to_datetime` switched to consistent datetime format parsing
+
+
+In the past, `to_datetime()` guessed the format for each element independently. This was appropriate for some cases where elements had mixed date formats - however, it would regularly cause problems when users expected a consistent format but the function would switch formats between elements. As of this version, parsing will use a consistent format, determined by the first non-NA value (unless the user specifies a format, in which case that is used).
```

Comment:
Just to confirm, `to_datetime` doesn't support a `format="fixed"` keyword right?

Reply:
Right, removed this section.